sktime/forecasting/base/adapters/_statsmodels.py

```diff
@@ -123,6 +123,88 @@ def _predict(self, fh, X=None):
         y_pred.name = self._y.name
         return y_pred
 
+    @staticmethod
+    def _extract_conf_int(prediction_results, alpha) -> pd.DataFrame:
+        """Construct confidence interval at specified `alpha` for each timestep.
+
+        Parameters
+        ----------
+        prediction_results : PredictionResults
+            results class for predictions
```

Comment:
I would be more specific here, "as returned by `self._fitted_forecaster.get_prediction`" ?

Reply:
Done.